bin/mapbox-gl-style-build

```diff
@@ -68,6 +79,19 @@ const buildStyles = async (styleDir, layerDir, outputDir, options = {}) => {
       process.exit(1);
     }
   });
+
+  const neverUsedVars = [...unusedContext].reduce((acc, path) => {
+    if (usedContext.has(path)) return acc;
```

Comment:
That's fair. I'm wondering if this could be simpler expressed as a filter? Like,

```js
const neverUsedVars = unusedContext.filter(path => !usedContext.has(path));
```

Or am I missing something?